### ISSUE TITLE
wrong position when pinning with masks

### ISSUE THREAD

**Issue by m-reiche** [matt-curtis/Fluid-for-Sketch#132]
Let's say you have group contining 2 other groups. The second group contains anything (just to be there). The first group is pinned to previous sibling, top 0 px, left 100% and contains a square mask 200 x 200 px and a second visible square 100 x 100 px.

As long as the small square (or any other visible item) is inside the mask everything is fine. The first group is positioned right next to the first group.

Now move the small square 100 px outsinde to the left of the mask. On update the group will now be moved about that amount of pixels to the left overlapping the first group. (which I do not understand. I'd understand if it would be orientated at the part outside the mask therfore goning more to the right.) Same is happening when the small square is going topwards out of the mask.

When you have no pinning activated the group wil move to top left endlessly with every update.

![bildschirmfoto 2017-11-10 um 16 31 36](https://user-images.githubusercontent.com/33550924/32665481-aed08f4e-c634-11e7-9593-a670de8d87f1.png)

**Comment by dinovidakovic** [comment]
Yup, masks are a problem, making it impossible for me to continue using this otherwise great plugin. Please let us know once this problem has been fixed. Thank you!
